includes/class-sensei-autoloader.php

```diff
@@ -206,6 +206,8 @@ public function initialize_class_file_map() {
 			'Sensei\Blocks\Course_Theme\Prev_Lesson'      => 'blocks/course-theme/class-prev-lesson.php',
 			'Sensei\Blocks\Course_Theme\Next_Lesson'      => 'blocks/course-theme/class-next-lesson.php',
 			'Sensei\Blocks\Course_Theme\Quiz_Button'      => 'blocks/course-theme/class-quiz-button.php',
+			'Sensei\Blocks\Course_Theme\Quiz_Back_To_Lesson' => 'blocks/course-theme/class-quiz-back-to-lesson.php',
+			'Sensei\Blocks\Course_Theme\Course_Progress_Counter' => 'blocks/course-theme/class-course-progress-counter.php',
```

Comment:
I see the file is not equally indented, isn't there any automatic formatting rules that helps with this?

Reply:
'Helps' is a strong word, yeah, the PHPCS linter usually harasses us with this, not sure why it didn't pick this one up.
(Also it's a bit special because it tries to only run for new code)

Reply:
That was a strong case that I also saw other times in our codebase.

Initially, I tried to indent properly, but when everything was indented, it was adding a warning. Not sure if there are some special checks for maximum sizes of properties or something like that. 🤷‍♂️